website/cue/reference/remap/functions/sieve.cue

```diff
@@ -0,0 +1,72 @@
+package metadata
+
+remap: functions: sieve: {
+	category: "String"
+	description: """
+		Keeps only mathces of `pattern` in `value`.
```

Comment:
```suggestion
		Keeps only matches of `pattern` in `value`.
```

Reply:
Thanks!
Although, I wonder how the spellchecker didn't catch this.